easybuild/easyconfigs/a/artic-ncov2019/artic-ncov2019-2021.06.21-foss-2020b.eb

```diff
@@ -0,0 +1,143 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+# Updated to latest version 24. Jun 2021 and bumped to foss-2020b
+# to use it with pangolin
+# J. Sassmannshausen (GSTT/NHS UK)
+
+easyblock = 'PythonBundle'
+
+name = 'artic-ncov2019'
+local_commit = 'b3f2dda5e6d95bc1c5d95a04d4ef37d304479477'
+version = '2021.06.21'
+
+homepage = "https://github.com/artic-network/artic-ncov2019"
+description = """Initial implementation of an ARTIC bioinformatics platform 
+for nanopore sequencing of nCoV2019 novel coronavirus."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),  # pandas,
+    ('Biopython', '1.78'),
+    ('Pysam', '0.16.0.1'),
+    ('BWA', '0.7.17'),
+    ('Eigen', '3.3.4', '', True),
+    ('minimap2', '2.20'),
+    ('SAMtools', '1.12'),
+    ('MUSCLE', '3.8.31'),
+    ('ETE', '3.1.2'),
+    ('MAFFT', '7.475', '-with-extensions'),
+    ('IQ-TREE', '2.1.2'),
+    ('snakemake', '6.1.0'),
+    ('Longshot', '0.4.3'),
+    ('medaka', '0.11.4'),
+    ('python-parasail', '1.2.4'),
+    ('PhyML', '3.3.20190321'),
+    ('nodejs', '12.19.0'),
+    ('goalign', '0.3.2', '', True),
+    ('gotree', '0.4.0', '', True),
+    ('rampart', '1.2.0rc3'),
+    ('libdeflate', '1.5'),
+    ('nanopolish', '0.13.3'),
+    ('seqtk', '1.3'),
+    ('BCFtools', '1.9'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('zipp', '1.0.0', {
+        'checksums': ['d38fbe01bbf7a3593a32bc35a9c4453c32bc42b98c377f9bff7e9f8da157786c'],
+    }),
+    ('pluggy', '0.13.1', {
+        'checksums': ['15b2acde666561e1298d71b523007ed7364de07029219b604cf808bfa1c765b0'],
+    }),
+    ('py', '1.8.1', {
+        'checksums': ['5e27081401262157467ad6e7f851b7aa402c5852dbcb3dae06768434de5752aa'],
+    }),
+    ('wcwidth', '0.1.9', {
+        'checksums': ['ee73862862a156bf77ff92b09034fc4825dd3af9cf81bc5b360668d425f3c5f1'],
+    }),
+    ('attrs', '19.3.0', {
+        'modulename': 'attr',
+        'checksums': ['f7b7ce16570fe9965acd6d30101a28f62fb4a7f9e926b3bbc9b61f8b04247e72'],
+    }),
+    ('importlib-metadata', '1.6.0', {
+        'modulename': 'importlib_metadata',
+        'source_tmpl': 'importlib_metadata-%(version)s.tar.gz',
+        'checksums': ['34513a8a0c4962bc66d35b359558fd8a5e10cd472d37aec5f66858addef32c1e'],
+    }),
+    ('more-itertools', '8.2.0', {
+        'modulename': 'more_itertools',
+        'checksums': ['b1ddb932186d8a6ac451e1d95844b382f55e12686d51ca0c68b6f61f2ab7a507'],
+    }),
+    ('packaging', '14.5', {
+        'checksums': ['363f9193daa14085b8dfeeb2bf64227bcf1dc85c02ae2a5c6018b01f77e46491'],
+    }),
+    ('args', '0.1.0', {
+        'checksums': ['a785b8d837625e9b61c39108532d95b85274acd679693b71ebb5156848fcf814'],
+    }),
+    ('pytest', '5.4.1', {
```

Comment:
Have you checked if all of those are really required? I remember pytest and attrs being part of Python ECs

You can us my findPythonDeps script from https://gist.github.com/Flamefire/49426e502cd8983757bd01a08a10ae0d

Reply:
I basically used `artic-ncov2019-2020.04.13-foss-2018b-Python-3.6.6.eb` as the base for that to have a guide. 
I will have a look at your findPythonDeps script and remove what is already there. I done some cleanup already, things which are already in `Rampart` and `TF` I have removed. I have also updated all the versions to the latest one. So, some bits I have already looked into independently.

Reply:
Update: I used your script and checked against `TF` and `medaka` and found one Python module which is already provided by `medaka`. So I removed that one.